examples/kuzu/Papers-100M/train.py

```diff
@@ -0,0 +1,150 @@
+import multiprocessing as mp
+import time
+
+import kuzu
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+import torch.sparse
+
+from torch_geometric.loader import NeighborLoader
+from torch_geometric.nn.conv import SAGEConv
+
+NUM_EPOCHS = 1
+LOADER_BATCH_SIZE = 1024
+
+print("Batch size:", LOADER_BATCH_SIZE)
+print("Number of epochs:", NUM_EPOCHS)
+
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+print('Using device:', device)
+
+# Load the train set
+train_df = pd.read_csv('papers100M-bin/split/time/train.csv.gz',
+                       compression='gzip', header=None)
+input_nodes = torch.tensor(train_df[0].values, dtype=torch.long)
+
+########################################################################
+# The below code sets up the remote backend of Kùzu for PyG.
+# Please refer to: https://kuzudb.com/docs/client-apis/python-api/overview.html
+# for how to use the Python API of Kùzu.
+########################################################################
+# The buffer pool size is set to 40GB. You can change it to a smaller value
+# if you have less memory.
+KUZU_BM_SIZE = 40 * 1024**3
+# Create kuzu database
+db = kuzu.Database("papers100M", KUZU_BM_SIZE)
+# Get remote backend for torch geometric
+feature_store, graph_store = db.get_torch_geometric_remote_backend(
+    mp.cpu_count())
+
+# Plug the graph store and feature store into the NeighborLoader
+loader_kuzu = NeighborLoader(
+    data=(feature_store, graph_store), num_neighbors={
+        ('paper', 'cites', 'paper'): [30, 30]
+    }, batch_size=LOADER_BATCH_SIZE, input_nodes=('paper', input_nodes),
+    input_time=None, num_workers=4)
```

Comment:
```suggestion
    input_time=None, num_workers=4, )
```
for nicer formatting.